### PR TITLE
v2.0.0

### PR DESCRIPTION
**Pull request by NorseGaud**

### CODE REVIEW

**Review by NorseGaud — COMMENTED** [review]
https://github.com/veertuinc/packer-builder-veertu-anka/blob/796e1988bfa1835c8443d852c3435f82f6116eaf/post-processor/ankaregistry/post-processor.go#L109

We should avoid setting a default here. Using --remote-vm is for very specific situations and will cause problems.

**Comment by tduffield** [comment]
I'm going to tweak things to avoid cloning the source VM if the target VM already exists on the machine (or the registry if `always_pull` is enabled).

File: builder/anka/communicator.go [hunk 1]
```diff
@@ -176,94 +160,112 @@ func (c *Communicator) UploadDir(dst string, src string, exclude []string) error
...
+	}
+
+	return c.Client.Copy(client.CopyParams{
+		Src: c.VMName + ":" + src,
+		Dst: dst,
+	})
+}
+
+func (c *Communicator) configureAnkaCP() {
+	if !c.Config.UseAnkaCP {
```
Comment (NivKeidan) [inline builder/anka/communicator.go:256]: Is there a need to call this code more than once?

Comment (NorseGaud) [reply builder/anka/communicator.go:256]: I do think a singleton would be useful here. I can work on that

Comment (NorseGaud) [reply builder/anka/communicator.go]: Merged in new logic to solve this

**Pull request merged**